gossip/src/cluster_info.rs

```diff
@@ -3082,7 +3090,7 @@ impl Node {
             )
             .unwrap();
         let tpu_forwards_quic =
-            bind_more_with_config(tpu_forwards_quic, QUIC_ENDPOINTS, quic_config).unwrap();
+            bind_more_with_config(tpu_forwards_quic, DEFAULT_QUIC_ENDPOINTS, quic_config).unwrap();
```

Comment:
What if we decided to change DEFAULT_QUIC_ENDPOINTS to a different value than 1?

Reply:
It should work with DEFAULT_QUIC_ENDPOINTS > 1. This code is used only for testing.

Reply:
Ditto.